src/state/slices/txHistorySlice/selectors.ts

```diff
@@ -127,6 +132,50 @@ export const selectTxIdsByFilter = createCachedSelector(
     : 'txIdsByFilter',
 )
 
+export const selectReceivedTxsForAccountIdsByFilter = createCachedSelector(
+  selectTxIds,
+  selectTxs,
+  selectWalletTxsByAccountIdAssetId,
+  (_state, filter: { accountIds: AccountId[]; txSender: string }) => filter.accountIds,
+  (_state, filter: { accountIds: AccountId[]; txSender: string }) => filter.txSender,
```

Comment:
Opted to go with inline selectors here instead of creating more filter selectors since these are only used once in this selector.